src/panels/lovelace/editor/conditions/types/ha-card-condition-user.ts

```diff
@@ -0,0 +1,113 @@
+import "@material/mwc-list";
+import { html, css, LitElement, PropertyValues } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import memoizeOne from "memoize-one";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import { stringCompare } from "../../../../../common/string/compare";
+import "../../../../../components/ha-list-item";
+import "../../../../../components/ha-switch";
+import "../../../../../components/user/ha-user-badge";
+import { fetchUsers, User } from "../../../../../data/user";
+import type { HomeAssistant } from "../../../../../types";
+import { UserCondition } from "../../../common/validate-condition";
+
+@customElement("ha-card-condition-user")
+export class HaCardConditionUser extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public condition!: UserCondition;
+
+  @property({ type: Boolean }) public disabled = false;
+
+  public static get defaultConfig(): UserCondition {
+    return { condition: "user", users: [] };
+  }
+
+  @state() private _users: User[] = [];
+
+  private _sortedUsers = memoizeOne((users: User[]) =>
+    users.sort((a, b) =>
+      stringCompare(a.name, b.name, this.hass.locale.language)
+    )
+  );
+
+  protected async firstUpdated(changedProps: PropertyValues) {
+    super.firstUpdated(changedProps);
+    this._fetchUsers();
+  }
+
+  private async _fetchUsers() {
+    const users = await fetchUsers(this.hass);
+    this._users = users.filter((user) => !user.system_generated);
+  }
+
+  protected render() {
+    const selectedUsers = this.condition.users ?? [];
+
+    return html`
+      <mwc-list>
+        ${this._sortedUsers(this._users).map(
+          (user) => html`
+            <ha-list-item graphic="avatar" hasMeta noninteractive>
```

Comment:
Or use `ha-check-list-item`?

Reply:
I updated to `ha-check-list-item` but I will look if we can have a user selector.